05_stringer/smartshark27/main_test.go

```diff
@@ -0,0 +1,21 @@
+package main
+
+import (
+	"bytes"
+	"strconv"
+	"testing"
+)
+
+func TestMainOutput(t *testing.T) {
+	var buf bytes.Buffer
+	out = &buf
+
+	main()
+
+	expected := strconv.Quote("127.0.0.1\n")
```

Comment:
You should consider a bigger set of tests, edge cases and also this is a perfect case to use table driven tests: https://dave.cheney.net/2019/05/07/prefer-table-driven-tests